src/components/calcite-time-picker/calcite-time-picker.scss

```diff
@@ -0,0 +1,154 @@
+:host {
+  @apply text-color-1
+    inline-block
+    font-medium
+    select-none;
+}
+
+.time-picker {
+  @apply shadow-2
+    items-center
+    flex;
+  border-radius: var(--calcite-border-radius);
+}
+
+span {
+  @apply items-center
+    bg-foreground-1
+    inline-flex
+    justify-center;
+
+  &.button {
+    @apply cursor-pointer;
+    &:hover,
+    &:focus {
+      @apply bg-foreground-2;
+    }
+    &:focus {
+      @apply outline-none;
+    }
+    &:active {
+      @apply bg-foreground-3;
+    }
+    &.top-left {
+      border-top-left-radius: var(--calcite-border-radius);
+    }
+    &.bottom-left {
+      border-bottom-left-radius: var(--calcite-border-radius);
+    }
+    &.top-right {
+      border-top-right-radius: var(--calcite-border-radius);
+    }
+    &.bottom-right {
+      border-bottom-right-radius: var(--calcite-border-radius);
+    }
+    calcite-icon {
+      @apply text-color-3;
+    }
+  }
+
+  &.input {
+    @apply font-medium relative;
+    &:hover {
+      box-shadow: inset 0 0 0 2px var(--calcite-ui-foreground-2);
+    }
+    &:focus,
+    &:hover:focus {
+      @apply outline-none;
+      box-shadow: inset 0 0 0 2px var(--calcite-ui-brand);
+    }
+  }
+}
+
+:host([scale="s"]) {
+  font-size: 14px;
```

Comment:
Should we be using pixel width's and font sizes? I'll leave the CSS to the professionals though :)

Reply:
updated these to use the tailwind `text-` scales.